Update performance benchmarks (#3818)
dapr/docs: daprdocs/content/en/operations/performance-and-scalability/perf-actors-activation.md

Reviewer: Is there a link to the Actor performance test that we can include, so that people can try this themselves?

* Sidecar Injector (control plane)
* Sentry (optional, control plane)

## Performance summary for Dapr v1.12

> The actors API in Dapr sidecar will identify which hosts are registered for a given actor type and route the request to the appropriate host for a given actor ID. The host runs an instance of the application and uses the Dapr SDK (.Net, Java, Python or PHP) to handle actors requests via HTTP.

Reviewer: ```suggestion
The actors API in Dapr sidecar identifies which hosts are registered for a given actor type and routes the request to the appropriate host for a given actor ID. The host runs an instance of the application and uses the Dapr SDK (.Net, Java, Python, Go) to handle actors requests via HTTP.
```
